Resources/Maps/frontier.yml

```diff
@@ -28,24 +28,11 @@ tilemap:
 entities:
 - proto: ""
   entities:
-  - uid: 1578
-    components:
-    - type: MetaData
-    - type: Transform
-    - type: Map
-    - type: PhysicsMap
-    - type: Broadphase
-    - type: OccluderTree
-    - parallax: FrontierStation
-      type: Parallax
-    - type: LoadedMap
-    - type: GridTree
-    - type: MovedGrids
```

Comment:
did you save it as a grid and not a map?

Reply:
Welp,
I did out of habit
Ill redo it from the same file again to not have anything screwup

Reply:
Fixed